edx/edx-ora2
# [BD-05] [BB-2992] [TNL-7578] Fix ORA grade explanation bug

We found a bug where sometimes the grade explanation message on an ORA block would say that the grade is determined by the staff grade, even though staff grading was disabled on that particular block

**JIRA tickets**: TNL-7578

**Merge deadline**: None

**Testing instructions**:

1. Checkout this PR & install it on your local devstack
2. Make sure the grade explanation message now shows the correct response for all assessment types (Self Assessment, Peer Assessment, Staff Assessment)
3. Try overriding the grade using staff controls, and recheck that the grade explanation message now says Staff Grade, regardless of the assessment type

**Author notes and concerns**:

1. I am curious about why this bug occured on my master devstack but not the juniper devstack

**Reviewers**
